dds/DdsDynamicData.idl

```diff
@@ -161,6 +161,16 @@ module DDS {
 
   local interface DynamicData;
 
+  local interface DynamicDataFactoryInterf {
+    // Implemented in C++
+    //static DynamicDataFactory get_instance();
+    // Not Implemented
+    //static DDS::ReturnCode_t delete_instance();
+    DynamicData create_data(in DynamicType type);
+    // Not Implemented
+    //static DDS::ReturnCode_t delete_instance(in DynamicData data);
```

Comment:
```suggestion
    DDS::ReturnCode_t delete_data(in DynamicData data);
```

This is the same pattern used in other factories, for example create_publisher/delete_publisher in the participant.

Reply:
Yes, but what would it do?

Reply:
Even if it's a no-op, it implements the spec requirements.